system/lib/libcxxabi/src/cxa_exception_emscripten.cpp

```diff
@@ -1,21 +1,30 @@
-//===------------------- cxa_exception_emscripten.cpp ---------------------===//
+//===------------------------- cxa_emscripten.cpp -------------------------===//
```

Comment:
This filename looks wrong.

Reply:
That's what you called it in your version: https://github.com/emscripten-core/emscripten/commit/3efb14451275d1806a0cb51ba6c9efa011a79560#diff-b09ab8d2f1ceaab1178f47623c925047cfec30a707e60e95d2ded9f3b7e70a14 (this is the link to your original commit)

Do you want me to change it?

Reply:
I mean the comment is wrong..  but yet it was wrong in my original PR :)

Reply:
Do you mean just the filename or is there anything else wrong here? I didn't pay much attention to parts that rebased cleanly, but happy to fix them too while I'm here, just let me know what 😅

Reply:
I just mean the name of the file in comment doesn't match the name of the file itself.